content/applications/finance/accounting.rst

```diff
@@ -82,9 +82,12 @@ Multi-company
 =============
 
 Several companies can be managed within the same database. Each company has its :doc:`chart of
-accounts <accounting/get_started/chart_of_accounts>`, which is also useful to generate consolidation
-reports. Users can access several companies but can only work on a single company's accounting at a
-time.
+accounts <accounting/get_started/chart_of_accounts>`, but it is possible to share accounts
+between them for scenarios in which such a configuration would be required.
```

Comment:
Shouldn't we also keep "Users can access several companies but can only work on a single company's accounting at a
time.", from the previous version of that paragraph ? :thinking: